From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: hopeless-ponderer <hopelessponderer1123@gmail.com>
Date: Mon, 19 Jul 2021 23:34:44 -0700
Subject: [PATCH] changed Carthaginian defection tech to switch 50% of enemy
 cities

both Carthaginian defection tech and Seleucid marriage pact take 60 seconds of research time (up from 40 seconds)
---
 .../public/simulation/data/technologies/defection.json    | 8 ++++----
 .../simulation/data/technologies/marriage_pact.json       | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/binaries/data/mods/public/simulation/data/technologies/defection.json b/binaries/data/mods/public/simulation/data/technologies/defection.json
index e124692f04..c660167c46 100644
--- a/binaries/data/mods/public/simulation/data/technologies/defection.json
+++ b/binaries/data/mods/public/simulation/data/technologies/defection.json
@@ -12,8 +12,8 @@
 	},
 	"requirementsTooltip": "Unlocked in Empire Phase.",
 	"icon": "sibylline_books.png",
-	"researchTime": 40,
-	"tooltip": "Gain control over 1/3 of your enemies' civil centers (selected randomly).",
+	"researchTime": 60,
+	"tooltip": "Gain control over 1/2 of your enemies' civil centers (selected randomly).",
 	"triggers": [{
 		"systemComponent": "Trigger",
 		"func": "TechTransferOwnershipEntities",
@@ -21,8 +21,8 @@
 			"receivingPlayerID": "{{ playerID }}",
 			"givingPlayersDiplo": ["Enemy"],
 			"entityClasses": ["CivCentre"],
-			"ratio": 0.33,
-			"minEntities": 3
+			"ratio": 0.5,
+			"minEntities": 2
 		}]
 	}],
 	"soundComplete": "interface/complete/building/complete_wonder.xml"
diff --git a/binaries/data/mods/public/simulation/data/technologies/marriage_pact.json b/binaries/data/mods/public/simulation/data/technologies/marriage_pact.json
index ea2c6de92d..9603faf0cb 100644
--- a/binaries/data/mods/public/simulation/data/technologies/marriage_pact.json
+++ b/binaries/data/mods/public/simulation/data/technologies/marriage_pact.json
@@ -21,7 +21,7 @@
 	},
 	"requirementsTooltip": "Unlocked in Empire Phase. Requires training the hero Seleucus I (Nikator).",
 	"icon": "sibylline_books.png",
-	"researchTime": 40,
+	"researchTime": 60,
 	"tooltip": "Receive a one-time gift of 20 Maurya War Elephants, in exchange for reduced territory push for Civil Centers.",
 	"modifications": [
 		{ "value": "TerritoryInfluence/Radius", "multiply": 0.7, "affects": "CivCentre" }
-- 
2.25.1

